### PR TITLE
Fix #217 and other misc updates

### LINKED ISSUE #217: [Bug] Typo in example Resource Records table for `svc` value
I noticed a typo [here](https://github.com/TBD54566975/did-dht-method/blob/d15960b9dc97de2bd1773e8533fed0ced0e365dc/spec/spec.md?plain=1#L477)

Currently the `Rdata` is:

```md
v=0;vm=k0,k1;auth=k0,k1;asm=k0,k1;inv=k0;del=k0;svc=s1
``` 

But should be:

```md
v=0;vm=k0,k1;auth=k0,k1;asm=k0,k1;inv=k0;del=k0;svc=s0
```

Notice the `svc=s1` vs (the correctly zero-indexed) `svc=s0`

### PR DESCRIPTION
**Pull request by thereisnogabe**
Fix #217
thank you @KendallWeihe

### CODE REVIEW

File: spec/spec.md [hunk 1]
```diff
@@ -696,15 +703,16 @@ One such registry is [provided by this specification](registry/index.html#gatewa
...
+A [[ref:Retained DID Set]] refers to the set of DIDs a [[ref:Gateway]] retains and [[ref:republishes]] to the DHT. A
 [[ref:Gateway]] may choose to surface additional [APIs](#gateway-api) based on this set, such as providing a
 [type index](#type-indexing).
 
 To safeguard equitable access to the resources of [[ref:Gateways]], which are publicly accessible and potentially
 subject to [a high volume of requests](#rate-limiting), we suggest an ****OPTIONAL**** mechanism aimed at upholding
-fairness in the retention and republishing of record sets by [[ref:Gateways]]. This mechanism, referred to as a
-[[ref:Retention Proof]], requires clients to generate a proof value for write requests. This process guarantees that
-the amount of work done by a client is proportional to the duration of data retention and republishing a [[ref:Gateway]]
+fairness in the retention and [[ref:republishing]] of record sets by [[ref:Gateways]]. This mechanism, referred to as a
```
Comment (thereisnogabe) [inline spec/spec.md:709]: note much of this language will change in a follow-up PR for #74

File: spec/spec.md [hunk 2]
```diff
@@ -19,6 +19,7 @@ The DID DHT Method Specification 1.0
 ~ [Moe Jangda](https://github.com/mistermoe)
 ~ [Frank Hinek](https://github.com/frankhinek)
 ~ [Henry Tsai](https://github.com/thehenrytsai)
+~ [Kendall Weihe](https://github.com/KendallWeihe)
```
Comment (KendallWeihe) [inline spec/spec.md:22]: hey that's me

File: spec/spec.md [hunk 3]
```diff
@@ -253,7 +260,7 @@ Additionally:
 
   - The `vm` property ****MUST**** always contain _at least_ the [[ref:Identity Key]] represented by `k0`.
 
-  - Verification Relationships (`auth`, `asm`, `inv`, `del`, `svc`) without any members ****MUST**** be omitted.
+  - Verification Relationships (`auth`, `asm`, `agm`, `inv`, `del`) any members ****MUST**** be omitted.
```
Comment (KendallWeihe) [inline spec/spec.md]: > any members ****MUST**** be omitted

What does this mean? Did you accidentally remove `without`?

```suggestion
  - Verification Relationships (`auth`, `asm`, `agm`, `inv`, `del`) without any members ****MUST**** be omitted.
```

File: spec/spec.md (hunk 1 above)
Comment (KendallWeihe) [reply spec/spec.md:709]: you may want to include the reasoning being to defend against spam, and specifically write (or create/update) spam, idk just an idea

**Review by KendallWeihe — APPROVED** [review]
one comment on the potentially accidentally deleted `without`? ✅

File: spec/spec.md (hunk 1 above)
Comment (thereisnogabe) [reply spec/spec.md:709]: this paragraph has a link to the section (#rate-limiting) which does this

**Pull request merged**